source/fundamentals/monitoring/command-monitoring.txt

```diff
@@ -30,6 +30,11 @@ events created by the MongoDB deployment:
 .. literalinclude:: /code-snippets/monitoring/apm-subscribe.js
    :language: javascript
 
+.. note::
+
+   To trigger events, enable command monitoring for the ``client`` by
+   specifying the ``monitorCommands`` option to ``true``.
```

Comment:
issue: I think this would be more clear to state:

> Command monitoring is disabled by default. To enable command monitoring, pass the option ... to your MongoClient constructor